sdk/build.gradle

```diff
@@ -49,13 +49,11 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.0.2'
     implementation 'com.android.volley:volley:1.1.1'
     testImplementation 'junit:junit:4.12'
+    testImplementation 'org.json:json:20140107'
     androidTestImplementation 'androidx.test.ext:junit:1.1.0'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
 }
 
 repositories {
     mavenCentral()
-}
-
-apply from: 'https://raw.githubusercontent.com/nisrulz/JCenter/master/installv1.gradle'
-apply from: 'https://raw.githubusercontent.com/nisrulz/JCenter/master/bintrayv1.gradle'
```

Comment:
Is er een reden waarom deze verwijderd is? Momenteel kan ik hierdoor niet publishen (kan het wel manueel terug toevoegen, maar dat is omslachtiger)

Reply:
In our opinion this was not secure. This downloads code directly from nisrulz github. If he changed his repo our build would change as wel. If you really need them you could make it secure by adding those files to our repo, that way we are in control.

Reply:
Fair enough

Reply:
Copied over the files to the repo for full control and ownership. See: 10ef83a8adc569d642b009e254d4119ef0e0fed8